chore: Upgrade remaining bigtable samples - issue #1389

saranshdhingra opened the pull request.

saranshdhingra commented:
The extra tag removed out of the 19 deleted region tags(bigtable_filters_print) isn't used in any page in the docs for any language.

bshaffer commented on bigtable/src/filter_composing_chain.php:
```diff
@@ -48,20 +48,17 @@ function filter_composing_chain(
     ]);
 
     foreach ($rows as $key => $row) {
+        // helper function defined in https://cloud.google.com/bigtable/docs/samples/bigtable-reads-print
```
Nit, but let's use a complete sentence here to be absolutely crystal clear:
```suggestion
        // The "print_row" helper function is defined in https://cloud.google.com/bigtable/docs/samples/bigtable-reads-print
```

bshaffer commented on bigtable/src/quickstart.php:
```diff
@@ -24,40 +22,39 @@
+$columnFamilyId = 'cf1';
+$columnId = 'c1';
+// Get the Value from the Row, using the column_family_id and column_id
+
```
remove extra newline
```suggestion
```

bshaffer commented on bigtable/src/quickstart.php:
```diff
@@ -24,40 +22,39 @@
+// $instanceId = 'The Bigtable instance ID';
+// $tableId = 'The Bigtable table ID';
+
+
```
Remove extra newline
```suggestion
```

bshaffer reviewed (commented):
One suggestion (which applies to each sample), a few style nits, and this looks really good!

bshaffer commented on bigtable/src/filter_limit_cells_per_col.php:
```diff
@@ -0,0 +1,84 @@
+namespace Google\Cloud\Samples\Bigtable;
+
+/**
+ * Copyright 2019 Google LLC.
```
The copyright years need updated on all these files!

bshaffer commented on bigtable/src/filter_limit_cells_per_col.php:
```diff
@@ -0,0 +1,84 @@
+<?php
+
+namespace Google\Cloud\Samples\Bigtable;
```
Namespace should go just above the region tags, to be consistent with our other samples

bshaffer commented on bigtable/src/filter_limit_col_family_regex.php:
```diff
@@ -0,0 +1,84 @@
+use Google\Cloud\Bigtable\Filter;
+
+/**
+ * Create a limiting filter on a column family with a regex
```
nit: lets put a newline between the description and `@param` tags
```suggestion
 * Create a limiting filter on a column family with a regex
 *
```

bshaffer reviewed (changes requested):
Just a few nits I missed

saranshdhingra commented:
> Just a few nits I missed. Also, do we have a plan for the snippet bot errors?

@bshaffer 

Namespace position: Updated in all bigtable samples
Line b/w description and `@param`: Updated in all bigtable samples
Copyright year change: Changed only in the new files(read snippets, write snippets, filter files, new samples added) as I imagine it's only for new files.

The CL for the new region tag paths is ready and as soon as this PR is merged we can proceed with that like the read and write snippets splitting.
